frontend/packages/kubevirt-plugin/src/components/modals/scheduling-modals/tolerations-modal/tolerations-modal.tsx

```diff
@@ -95,6 +104,22 @@ export const TModal = withHandlePromise(
       <div className="modal-content">
         <ModalTitle>{TOLERATIONS_MODAL_TITLE}</ModalTitle>
         <ModalBody>
+          <div className="affinity-modal__desc-container">
```

Comment:
We should change the class name and file name move the scss file to the shared folder of scheduling modals.

Reply:
:+1: moved